### DISCUSSION TITLE
Codeium stopped working suddenly

### DISCUSSION THREAD

**Discussion by pannet1** [LazyVim/LazyVim#1473] [Q&A]
hi,
it was a great experience with LazyVim so far. I recently stumbled upon codeium and it was a match made in heaven. somehow recently it is not working and i am not able to update the token either. This is the error when i run `Codeium Auth` and paste the token.

![image](https://github.com/LazyVim/LazyVim/assets/12350276/2df67826-f55b-4375-b8bd-471afcd6bd6d)

can someone  give me an idea on why this is happening

**Comment by dpetka2001** [comment]
Your best bet would be to open an issue at the plugin's repo, as they are more knowledgeable about the plugin's internals.

**Reply by pannet1** [reply]
you are my saviour @dpetka2001
